include/deal.II/base/point.h

```diff
@@ -107,8 +106,7 @@ DEAL_II_NAMESPACE_OPEN
  * @author Wolfgang Bangerth, 1997
  */
 template <int dim, typename Number = double>
-class Point : public Tensor<1, dim, Number>
-{
+class Point : public Tensor<1, dim, Number> {
```

Comment:
There are now a large number of unrelated indentations. Please run the indentation script. (If you use the `./contrib/utilities/download_clang_format` script, it will download the right version for you. You don't have to install or compile it yourself.)

Reply:
I shall close this PR, work on a new branch and raise a new PR.